ksqldb-common/src/test/java/io/confluent/ksql/QueryApplicationIdTest.java

```diff
@@ -12,6 +12,7 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
+package io.confluent.ksql;
```

Comment:
the surefire plugin wasn't happy with this class not being part of any module

Reply:
Neither am I! Thanks for the fix.